exchange/bidder.go

```diff
@@ -379,14 +379,20 @@ func (bidder *bidderAdapter) requestBid(ctx context.Context, bidderRequest Bidde
 							}
 						}
 
+						alternateBidderCode := ""
+						if !strings.EqualFold(bidderRequest.BidderName.String(), bidderName.String()) {
```

Comment:
Is there a test in place where `bidderRequest.BidderName` is different from `bidderName.string`

I can't seem to find it, but just want to ensure we're testing the main case of this PR that you laid out.

Reply:
In case of aliase bidder its possible to have bidderRequest.BidderName is different that bidderName.
For example groupm is alias/alternate bidder code of pubmatic. So in this case bidderRequest.BidderName will be pubmatic and bidderName will be groupm. We are expecting adaptercode as pubmatic.

Reply:
I see, is it possible to add a json test of some kind where we can have this situation covered?

Reply:
Added json test file alternate-bidder-codes-aliase.json